### PR TITLE
Add OpenMetrics (sandbox)

### PR DESCRIPTION
**Pull request by RichiH**
Signed-off-by: Richard Hartmann <richih@richih.org>

### CODE REVIEW

**Comment by caniszczyk** [comment]
RFC @cncf/toc 

I'll leave this open for comment for a week or so

File: proposals/openmetrics.adoc [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+* Oath.com
+* RobustPerception
+* SpaceNet
+* Uber
+
+OpenMetrics was presented at the [CNCF TOC meeting on 2018-06-19](https://docs.google.com/presentation/d/1Ym8fLRCaX43uHPHBRyuRXM62U8m4vXaBXkuUp6tt3js/edit#slide=id.g25ca91f87f_0_0).
+
+*Statement on alignment with CNCF mission:*
+
+Given the CNCF's stated role in "fostering the growth and evolution of the cosystem" and "making the technology accessible and reliable", we believe OpenMetrics helps with both of these goals.
```
Comment (simonpasquier) [inline proposals/openmetrics.adoc]: ecosystem

File: proposals/openmetrics.adoc [hunk 2]
```diff
@@ -0,0 +1,77 @@
+== OpenMetrics
+
+*Name of project:* OpenMetrics
+
+*Description:*
+
+OpenMetrics refines the Prometheus exposition format into an independent standard.
+
+Prometheus has become the de facto standard in cloud-native metric monitoring, and has active upstream work by competitors.
+The ease of implementing this exposition data has lead to an explosion in compatible metrics endpoints with 300+ exporters registered, dozens of native integrations, and unknown numbers of internal adoptions.
```
Comment (simonpasquier) [inline proposals/openmetrics.adoc]: s/lead/led/ ?

**Comment by RichiH** [comment]
Fixed, thanks!

**Comment by RichiH** [comment]
@caniszczyk Two weeks have passed.

**Comment by caniszczyk** [comment]
@RichiH we will merge this in on August 9th for PromCon as requested

**Pull request merged**